app/controllers/imports_controller.rb

```diff
@@ -28,7 +28,7 @@ def create
 
     respond_to do |format|
       if @import.save
-        @import.import_later
+        @import.import_later(ActiveRecord::Type::Boolean.new.cast(params[:podcast_import][:import_metadata]))
```

Comment:
passing in a boolean here determines whether or not `import_podcast!` will be called.

Reply:
Minor nitpick - I might move that params parsing to another line / a private method, for readability.